supervisor/errata_utils.py

```diff
@@ -430,17 +481,32 @@ def get_erratum_build_nvr(erratum_id: str | int, package_name: str) -> str | Non
     return None
 
 
-def errata_have_same_file_lists(errata_id1: int | str, errata_id2: int | str):
-    """Check if the given errata have the same file lists
-
-    After stripping package versions and RHEL release versions,
-    do the two errata ship the same subpackages for each
-    variant and architecture?
+def generate_comment_for_product_listings_comparison_result(
```

Comment:
I'd have this function in erratum_handler - I'd like to keep errata_utils.py to be fairly generic API that is not specific to exactly what we are doing.